src/lib/slack/common.js

```diff
@@ -5,38 +5,31 @@ const simpleGit = require('simple-git/promise')(appRoot.path);
 const users = require('../users');
 
 async function generateAndSendBootMessage(channel = null, { msgText = null } = {}) {
-  const { available, benched } = await users.listAllUserNamesByAvailability();
-  const SHA = await simpleGit.revparse(['HEAD']);
+  const [{ availableUsers, benchedUsers }, SHA] = await Promise.all([
```

Comment:
I think there's an edge-case here.

it generated the initial boot message fine but then admins saying `overview` was giving an empty list back.